source/standards/programming-languages.html.md.erb

```diff
@@ -31,7 +31,11 @@ The service manual has information on
 [manual_js]: https://www.gov.uk/service-manual/technology/using-progressive-enhancement
 
 GDS projects that use Node.js include:
-GOV.UK Frontend, GOV.UK Pay, Performance Platform and GOV.UK Platform as a Service.
+
+- GOV.UK Frontend
+- GOV.UK Pay
+- Performance Platform
+- GOV.UK Platform as a Service (TypeScript)
```

Comment:
Verify are also using TypeScript in the form of https://github.com/alphagov/passport-verify, although this is a client library used by other bits of Government rather than something we run directly, so maybe it doesn't count (I can see a world where we provide PHP client libraries but don't support PHP as a GDS way language).